# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/qmatplot/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/ksmatrix.h.orig	2004-09-11 06:37:29.176089088 +0200
+++ ./src/ksmatrix.h	2004-09-11 06:37:57.477786576 +0200
@@ -332,8 +332,8 @@
 class KSMatrixFormula : public QSMatrix
  {
 	Q_OBJECT
-	Q_PROPERTY( QString formula READ formula WRITE setFormula );
-	Q_PROPERTY( bool transposition READ transposition WRITE setTransposition );
+	Q_PROPERTY( QString formula READ formula WRITE setFormula )
+	Q_PROPERTY( bool transposition READ transposition WRITE setTransposition )
 
   public:
 	/**
--- ./src/qscobjects.h.orig	2004-09-11 06:41:39.403048808 +0200
+++ ./src/qscobjects.h	2004-09-11 06:42:05.619063368 +0200
@@ -246,7 +246,7 @@
 	QSGArrow m_end2style;
 	QSGPoint m_end1point;
 	QSGPoint m_end2point;
-	QSGLine m_line;;
+	QSGLine m_line;
    };
 
 //-----------------------------------------------------------------------//
--- ./src/kspanelmanager.cpp.orig	2004-09-11 06:42:46.286880920 +0200
+++ ./src/kspanelmanager.cpp	2004-09-11 06:42:57.271211048 +0200
@@ -47,7 +47,7 @@
 	 Qt::Orientation	o;
 
          qmatplotShell  	*shell;
-	 KSWorkbook 		*workbook;;
+	 KSWorkbook 		*workbook;
 	 KSPanel 		*curr_panel;
 
 	 QSplitter		*splitter;
--- ./src/kswinworksheet.h.orig	2004-09-11 06:45:34.993233656 +0200
+++ ./src/kswinworksheet.h	2004-09-11 06:46:08.607123568 +0200
@@ -40,7 +40,7 @@
   public:
 	KSWinWorksheet( KSWorkspace *workspace, bool worksheetMode );
 	virtual ~KSWinWorksheet();
-	void KSWinWorksheet::setDataObject( QSData *data );
+	void setDataObject( QSData *data );
 	KSChannelList *chanList() const;
 	KSChannelListItem *selectedItem() const;
 	KSMPanel *settingsPanel() const;
